openedx/core/djangoapps/theming/helpers.py

```diff
@@ -25,24 +26,28 @@ def get_page_title_breadcrumbs(*args):
 
 def get_value(val_name, default=None, **kwargs):
     """
-    This is a proxy function to hide microsite_configuration behind comprehensive theming.
+    This is a proxy function to hide microsite_configuration and site_configuration behind comprehensive theming.
     """
 
-    # Retrieve the requested field/value from the microsite configuration
-    microsite_value = microsite.get_value(val_name, default=default, **kwargs)
+    if configuration_helpers.is_site_configuration_enabled():
```

Comment:
+1 to @mattdrayer's comment here. Also, it would be helpful if there was a function on the SiteConfiguration model that would do the setting override lookup for us, so that we could call it via:

`request.site.siteconfiguration.get_setting('PLATFORM_NAME')`